tests/test_workers_db_storage.py

```
@@ -208,15 +206,15 @@ def test_process_batch_stores_all_transcripts(
             episodes.append(episode)
 
         worker = TranscriptionWorker(config=mock_config, repository=repository)
-        
+
         # Mock transcription
         mock_model = MagicMock()
         def mock_transcribe(audio_path, *args, **kwargs):
```

Ruff (0.14.10)

Unused function argument: `args`

(ARG001)